src/main/frontend/app/containers/ReviewContainer.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component} from 'react'
+import TextField from '../components/TextField'
+import SelectField from '../components/SelectField'
+import TextArea from '../components/TextArea'
+
+class ReviewContainer extends Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            bodyType:'',
+            itemSize:'',
+            quality:'',
+            comment:'',
+            photo:'',
+            rating:''
+        }
+        this.handleChange = this.handleChange.bind(this)
+        this.handleSubmit = this.handleSubmit.bind(this)
+        this.handleClear = this.handleClear.bind(this)
+    }
+
+    handleChange(event){
+        this.setState({
+            [event.target.name]: event.target.value
+        })
+    }
+
+    handleSubmit(event){
+        event.preventDefault()
+        this.props.addNewReview(this.state)
+        this.handleClear()
+    }
+
+    handleClear(){
+        this.setState({
+            bodyType:'',
+            itemSize:'',
+            quality:'',
+            comment:'',
+            photo:'',
+            rating:''
+        })
+    }
+
+    render() {
+        let bodyTypeArray = ['straight','pear','oval', 'diamond']
+        let bodyTypeOptions = bodyTypeArray.map((name, i) => {
+            return <option key={i} value={i}>{name}</option>
+        })
+        let sizeArray = ['US 0', 'US 2', 'US 4', 'US 8', 'US 10', 'US 12', 'US 14', 'US 16', 'US 18', 'US 20']
+        let sizeOptions = sizeArray.map((name, i) => {
+            return <option key={i} value={i}>{name}</option>
+        })
```

Comment:
It's probably worth abstracting bodyTypeArray and sizeArray to their own constants.